src/napari_spatialdata/_interactive.py

```diff
@@ -77,7 +80,36 @@ def _onClick(self, text: str) -> None:
         elif self.elements_widget._elements[text] == "shapes":
             self._add_shapes(text)
 
+    def _update_visible_in_cs(self, event: Event) -> None:
+        """Toggle active in cs metadata when changing visibility of layer."""
+        layer = event.source
+        layer_active = layer.metadata["active_in_cs"]
+        selected_coordinate_system = self.coordinate_system_widget._system
+
+        elements = self.elements_widget._elements
+        if layer.name in elements:
+            if selected_coordinate_system not in layer_active:
+                layer_active.add(selected_coordinate_system)
+            else:
+                layer_active.remove(selected_coordinate_system)
+
+    def _update_layers_visibility(self) -> None:
+        """Toggle layer visibility dependent on presence in currently selected coordinate system."""
+        elements = self.elements_widget._elements
+        coordinate_space = self.coordinate_system_widget._system
```

Comment:
It would be nice if we could unify the usage of "coordinate system" and "coordinate space". I think we call them coordinate systems in SpatialData, but I'm not 100% sure. @LucaMarconato  can you confirm?

Reply:
True we use coordinate systems. Resolved with [2ec5203](https://github.com/scverse/napari-spatialdata/pull/99/commits/2ec5203d9858db9952e7375f5374d042a4e06738)